src/node.rs

```diff
@@ -598,10 +614,15 @@ impl PbftNode {
         peer_res
     }
 
+    pub fn force_view_change(&mut self, state: &mut PbftState) {
+        info!("{}: Forcing view change", state);
+        handlers::force_view_change(state, &mut *self.service)
```

Comment:
I would argue that we should start going the other way and move `handlers.rs` into this file. Yeah it's going to be a big file (though we can cut the size down a bit by moving the tests around), but splitting the definition of an object over multiple files just because it's big rubs me the wrong way.

Reply:
As far as I can tell right now, there is no clean separation of concerns between the engine, node, state, and handlers. This is as consistent with the existing organization as I could get without embarking on a large scale refactoring to better separate concerns. I'd be up for having a discussion about to do the reorganization.

Reply:
I was just thinking of having the logic over here (or keeping the logic over in `handlers` and calling `handlers::force_view_change` directly from other code instead of creating the `Node::force_view_change` proxy), not advocating for a large-scale refactoring right now.